### DISCUSSION TITLE
GELF logs in docker swarm from nginx

### DISCUSSION THREAD

**Discussion by JumpAttacker** [datalust/seq-tickets#1562] [Help]
Hey, im using guid from documantation. But i have messages with wrong unicode
![image](https://user-images.githubusercontent.com/10989729/166329622-95df2ad5-c047-4290-a312-02ef5d17e571.png)

And there is no info about my problemm

**Comment by KodrAus** [comment]
Hi @JumpAttacker :wave: The GELF app assumes text is encoded as UTF8. Are you able to share any more details about your environment? Are you using the GELF container? Or as a Seq app?

**Reply by JumpAttacker** [reply]
> 

Hey, it's already fixed. I just sent logs to seq, but not to seq GELF plugin. Thanks

**Reply by KodrAus** [reply]
No problem! Glad you got it sorted. It sounds like there was something funny going on there so if you do have a chance to share any details on how you were holding the GELF input I'd love to hear them :bow:
